azure-pipelines.yml

```diff
@@ -52,7 +52,7 @@ jobs:
   - script: |
       set -e
       brew update
-      brew install gnu-time opam pkg-config gtksourceview3
+      brew install gnu-time opam pkg-config gtksourceview3 adwaita-icon-theme
```

Comment:
Do we need to update install instructions somewhere?

Reply:
I don't think so, this package is there so that we can copy come icons in the bundle (that is supposed to work even if you don't have brew)